rabbit-escape-engine/src/rabbitescape/engine/Entrance.java

```diff
@@ -31,6 +34,7 @@ public void calcNewState( World world )
     @Override
     public void step( World world )
     {
+        
```

Comment:
Whitespace changes should be in separate commits (in fact, this one is probably not needed).